### ISSUE TITLE
Document the meaning of different strategies

### ISSUE THREAD

**Issue by gajus** [sanyuan0704/vite-plugin-chunk-split#10]
At the moment, it does not say what does `single-vendor` do.

**Comment by milahu** [comment]
strategy `single-vendor` is not implemented

https://github.com/sanyuan0704/vite-plugin-chunk-split/blob/b8504a141d6d13cab206777b3d6577531e2658a8/src/index.ts#L98-L153

**Comment by nathan-kerr** [comment]
Yes, please. It is not clear how to decide which strategies to use in different scenarios.
